actions/update-swversion.py

```diff
@@ -0,0 +1,69 @@
+"""Ran daily to automatically obtain and commit latest swversion."""
+
+import json
+import os
+import re
+
+import requests
+
+
+def load_json(filename: str) -> dict:
+    """Load JSON from file."""
+    try:
+        with open(filename, encoding="utf-8") as fdesc:
+            return json.loads(fdesc.read())  # type: ignore
```

Comment:
why the type ignore here ?

Reply:
So these two methods the load and save_json were both copied over from the main application with a slight modification to the save_json to support being used in this purpose. I had originally wanted to simply import our existing code instead of doing this copy/paste but realized the original code would need some modification (specifically the backup code) to support this. In effect, I avoided modifying the core code to support this as it didn't really make much sense. The downside, of course, is double maintenance. 

The type ignore here was present in the original code I copied.